apps/condo/domains/miniapp/schema/B2BAppNewsSharingConfig.test.js

```diff
@@ -0,0 +1,160 @@
+/**
+ * Generated by `createschema miniapp.B2BAppNewsSharingConfig 'publishUrl:Text; previewUrl:Text; getRecipientsUrl:Text;'`
+ */
+
+const { faker } = require('@faker-js/faker')
+
+const { makeLoggedInAdminClient, makeClient, expectValuesOfCommonFields } = require('@open-condo/keystone/test.utils')
+const {
+    expectToThrowAuthenticationErrorToObj, expectToThrowAuthenticationErrorToObjects,
+    expectToThrowAccessDeniedErrorToObj,
+} = require('@open-condo/keystone/test.utils')
+
+const { B2BAppNewsSharingConfig, createTestB2BAppNewsSharingConfig, updateTestB2BAppNewsSharingConfig } = require('@condo/domains/miniapp/utils/testSchema')
+const { makeClientWithNewRegisteredAndLoggedInUser, makeClientWithSupportUser } = require('@condo/domains/user/utils/testSchema')
+
+
+describe('B2BAppNewsSharingConfig', () => {
```

Comment:
Any direct access tests? Maybe in `UserRightSet` schema in `development-portal` section, since it's probably be filled from it service user